x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/string/BitLength.java

```diff
@@ -40,6 +40,7 @@ public class BitLength extends UnaryScalarFunction {
     @FunctionInfo(
         returnType = "integer",
         description = "Returns the bit length of a string.",
+        note = "All strings are in UTF-8, so a single character can use multiple bytes.",
```

Comment:
I see this useful addition, but it did not appear in the docs preview. That seems a bit odd.

Reply:
Updated the .docs files.